projects/experimental/components/label/label.directive.ts

```diff
@@ -0,0 +1,21 @@
+import {Directive, ElementRef, HostBinding, inject} from '@angular/core';
+import {tuiWithStyles} from '@taiga-ui/cdk';
+import {TuiTextfieldComponent} from '@taiga-ui/experimental/components/textfield';
+
+import {TuiLabelComponent} from './label.component';
+
+@Directive({
+    standalone: true,
+    selector: 'label[tuiLabel]',
+})
+export class TuiLabelDirective {
+    // @ts-ignore
+    private readonly nothing = tuiWithStyles(TuiLabelComponent);
```

Comment:
`nothing` is expected name?)

Reply:
Well, we can add constructor and run that code inside of it, as an alternative. I was also thinking of calling it void. There's no other way to run initialization code, unfortunately.